layouts/shortcodes/figure.html

```diff
@@ -7,7 +7,12 @@
 alt = 'Cute puppies'
 height = 200
 width = 200
-caption = 'A figure is an image with a caption and/or a legend:'
+title = 'Figure title'
+attr = '(**Figure Credits:** Daily cute puppy image from unslash.com)'
```

Comment:
Should we add the brackets programatically?

Reply:
We were thinking of another name... I think you suggested `credit` and `credit_link/url`? Otherwise `attribution` written out would work.

Reply:
I removed the example with parentheses and changed `attr` to `attribution`.